src/client/app/pages/SearchPage/data/searchPageActions.js

```diff
@@ -0,0 +1,61 @@
+import {fetchMoviesByTitle} from '../../../api/api';
+import {SORTS, SOURCE, SEARCH_ACTIONS} from './consts';
+
+export function searchMovies (query, sortBy) {
+  if (query) {
+    return (dispatch) => {
+      dispatch({
+        type: SEARCH_ACTIONS.SEARCH_MOVIE_REQUEST,
+        payload: query
+      });
+
+      fetchMoviesByTitle(query)
+        .then((movies) => {
+          dispatch({
+            type: SEARCH_ACTIONS.SEARCH_MOVIE_SUCCESS,
+            payload: sortMovies(movies, sortBy)
+          })
+        })
+        .catch(() => {
+          dispatch({
+            type: SEARCH_ACTIONS.SEARCH_MOVIE_ERROR,
+            payload: []
+          });
+        });
+    }
+  }
+  return {
```

Comment:
The same idea as in above case. No reason to send an action if any parameters aren't valid